tests/integration.rs

```diff
@@ -1,6 +1,13 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #![allow(dead_code)]
+#![cfg(not(feature = "gdb"))]
+
+#[cfg(all(not(miri), not(feature = "backend-nil")))]
+mod exec;
+#[cfg(all(not(miri), not(feature = "backend-nil")))]
```

Comment:
veto

Reply:
this removes testing if exec and syscalls if all features are enabled

Reply:
make that `not(target_os = "linux")` maybe?

Reply:
Elaborate?

Reply:
sorry this one:
```
#[cfg(all(target_os = "linux", target_arch = "x86_64"))]
```

Reply:
> this removes testing of exec and syscalls if all features are enabled

These tests are already disabled if all features are enabled, because of the not(feature = "gdb")) above. More concerning is that this is disabled under the default feature profile.